modules/game/src/main/GameRepo.scala

```diff
@@ -342,7 +342,7 @@ object GameRepo {
       Match($doc(F.playerUids -> $doc("$ne" -> userId))),
       GroupField(F.playerUids)("gs" -> SumValue(1)),
       Sort(Descending("gs")),
-      Limit(limit))).map(_.documents.flatMap { obj =>
+      Limit(limit))).map(_.firstBatch.flatMap { obj =>
```

Comment:
Same behaviour, but with the right naming, as indeed it get the first batch of result (with is in your cases the only one ... cursor support for aggregation result is there for the other possible cases)